### PR TITLE
add msgpack5 v3.0.0 with git autoupdate

### PR DESCRIPTION
**Pull request by maruilian11**

### CODE REVIEW

File: ajax/libs/msgpack5/package.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      "ipad/6",
+      "android-browser/latest"
+    ]
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/mcollina/msgpack5.git",
+    "basePath": "dist",
+    "files": [
+      "**"
```
Comment (PeterDaveHello) [inline ajax/libs/msgpack5/package.json]: Should be `**/*` not `**`.

**Comment by maruilian11** [comment]
fixed, pls take a look.

**Comment by PeterDaveHello** [comment]
Can you update commit message to some kind of this:
`add lightgallery v1.1.5 with git autoupdate, close #5017`

**Comment by maruilian11** [comment]
ok

**Pull request merged**